src/lib/common/files.ts

```diff
@@ -117,6 +117,7 @@ export function serializeSchemes(authority: string, gj: Schemes): Schemes {
   }
   copy.authority = authority;
   copy.origin = "atip-v2";
+  copy.sketch_exported = new Date().toDateString();
```

Comment:
I think for dates/datetimes probably best to try and go with ISO 8601 format to help with machine readability as well as human. Which I think you can achieve with [`Date.prototype.toISOString`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString). 

```suggestion
  copy.sketch_exported = new Date().toISOString();
```

Reply:
makes complete sense - I tried a couple and went for increased human readability, but given it's going into the data platform better to go for the more easily processed format